src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -455,7 +455,8 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
 
   test("Create a new app (that uses secrets) successfully") {
     Given("The secrets feature is enabled")
-    AllConf.withTestConfig(Seq("--enable_features", "secrets"))
+    configArgs = Seq("--enable_features", "secrets")
+    resetAppsResource
```

Comment:
Hm, does `configArgs` have to be mutable? It's probably too much for this diff but it feels that `configArgs`, `appsResource` etc are fixtures more or less unique to each test case.